docs/user_guide/capsules/isolation.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: documentation
+title: Capsule Isolation
+sidebar: sidebars/documentation.html
+---
+
+# Capsule Isolation
+
+The goal of Capsule Isolation is to provide a single endpoint for all of a client's communication, so that in remote network segments, you need only open Firewall ports to the Capsule itself.
+
+![Diagram of Communication](./isolation.png)
+
+## Content and Configuration Services
+
+There are 4 primary areas that require client communication:
+
+### Package Installation
+
+That is, yum.  Katello Capsules by default have the Pulp feature, which mirrors content for the selected Lifecycle Environments.  Enable the Content Mirror with `--pulp=true` on the capsule-installer. Clients need to be able to communicate with the Capsule on port 443/tcp.
+
+### Katello Agent
+
+The Katello agent is a goferd plugin that allows you to schedule remote actions on hosts such as package installation, updates, etc.  In order to use the Katello agent against a capsule, you will need to enable the Qpid Dispatch Router feature (`--qpid-router=true`) in the capsule-installer.  Clients need to be able to communicate with the Capsule on port 5467/tcp.
+
+### Puppet & Puppet CA
+
+By default, the Puppet CA feature on the Capsule is an indepedent CA which will manage the certificates for all the clients registered against the Capsule.  Simply select the Puppetmaster and Puppet CA to be the Capsule when creating a host.  Enable these features with `--puppet=true --puppetca=true`.  Clients need to communicate with the Capsule on port 8140/tcp.
```

Comment:
Should we mention that including a puppet ca is required to isolate puppet traffic currently?